operate/services/manage.py

```diff
@@ -947,7 +947,7 @@ async def healthcheck_job(
                 logging.info(
                     f"Error occured while checking the service health\n{traceback.format_exc()}"
                 )
-            await asyncio.sleep(300)
+            await asyncio.sleep(30)
```

Comment:
I think this constant should be defined somewhere

Reply:
@dvilelaf  good point, same time refactoring  - another task